packages/export-selector-ui/src/lib/components/ExportAccount.tsx

```diff
@@ -0,0 +1,351 @@
+};
+
+const ACCESS_KEY_TYPES = {
+  LEDGER: "Ledger",
+  FULL_ACCESS_KEY: "Full Access Key",
+  MULTI_SIG: "Multi-Sig",
+  UNKNOWN: "Unknown",
+};
+
+const permissionToType = (
+  permission: string | FunctionCallPermissionView
+): string => {
+  if (permission === "FullAccess") {
+    return ACCESS_KEY_TYPES.FULL_ACCESS_KEY;
+  }
+  //@ts-ignore
+  if (permission?.FunctionCall?.method_names.includes("__wallet__metadata")) {
+    return ACCESS_KEY_TYPES.LEDGER;
+  }
+  const multiSigMethods = [
+    "add_request",
+    "add_request_and_confirm",
+    "delete_request",
+    "confirm",
+  ];
+  if (
+    //@ts-ignore
+    permission?.FunctionCall?.method_names.every((method: string) =>
+      multiSigMethods.includes(method)
+    )
+  ) {
+    return ACCESS_KEY_TYPES.MULTI_SIG;
+  }
+  return ACCESS_KEY_TYPES.UNKNOWN;
+};
+
+interface getAccountTypeProps {
+  provider: providers.Provider;
+  accountId: string;
+  publicKey: string;
+}
+
+const getAccountType = async ({
+  provider,
+  accountId,
+  publicKey,
+}: getAccountTypeProps) => {
+  const result = await provider
+    .query<AccessKeyView>({
+      request_type: "view_access_key",
+      account_id: accountId,
+      public_key: publicKey,
+      finality: "final",
+    })
+    .then((data) => {
+      const type = permissionToType(data.permission);
+      return {
+        type,
+      };
+    })
+    .catch(() => {
+      return {
+        type: ACCESS_KEY_TYPES.UNKNOWN,
+      };
+    });
+  return result;
+};
+
+interface getAccountBalanceProps {
+  provider: providers.Provider;
+  accountId: string;
+}
+
+const getAccountBalance = async ({
+  provider,
+  accountId,
+}: getAccountBalanceProps) => {
+  const result = await provider
+    .query<AccountView>({
+      request_type: "view_account",
+      finality: "final",
+      account_id: accountId,
+    })
+    .then((data) => {
+      const bn = new BN(data.amount);
+      return {
+        hasBalance: !bn.isZero(),
+      };
+    });
+  return result;
+};
+
+export interface ExportAccountData {
+  accountId: string;
+  privateKey: string;
+  hasBalance: boolean;
+  type: string;
+}
+
+export const ExportAccount: React.FC<ExportAccountProps> = ({
+  alertMessage,
+  module,
+  onCloseModal,
+  onWarning,
+  accounts,
+  selector,
+  wallet,
+  onBack,
+  onComplete,
+}) => {
+  const [selectedAccounts, setSelectedAccounts] = useState<Array<string>>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [accountsWithDetail, setAccountsWithDetail] = useState<
+    Array<ExportAccountData>
+  >([]);
+  const [disabledAccounts, setDisabledAccounts] = useState<
+    Array<ExportAccountData>
+  >([]);
+
+  //@ts-ignore
+  const [exportInterfaces, setExportInterfaces] = useState<{
+    buildImportAccountsUrl?: BrowserWalletBehaviour["buildImportAccountsUrl"];
+    importAccountsInSecureContext?: InjectedWalletBehaviour["importAccountsInSecureContext"];
+    // TODO:  once WEP-219 is completed, implement following interface
```

Comment:
What would you think about leaving out QR code references until we're ready to implement the interface?

Reply:
you mean remove them from this PR?
I have implemented them with assumption that we will implement QR code interfaces soon. But if we can't give certainty at this stage, I think it is wise to delete them to avoid confusion. 

If we are removing QR code references, do you prefer to remove them completely or just in this component?